react/react-styled-components/react-styled-components_box/pages/index.js

```diff
@@ -1,10 +1,18 @@
+import styled from "styled-components";
```

Comment:
You don't need this import here. `import StyledBox` this will be enough

Reply:
Hi @ Ruslan, I checked it again, but I don't understand how else you want to do it?

I imported this because of the bonus task. So I created this outer div, to set all boxes to display: flex.

**Bonus: Aligning Boxes Horizontally

Using styled components, can you align the four boxes horizontally by only changing ./pages/index.js?

    Hint: Use display: flex to achieve this goal.**